scripts/regenerate.sh

```diff
@@ -31,7 +28,7 @@ mkdir -p "${GOBIN}"
 echo "remove existing generated files"
 # grpc_testing_not_regenerate/*.pb.go is not re-generated,
 # see grpc_testing_not_regenerate/README.md for details.
-rm -f $(find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate')
+find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate' | xargs rm -f || true
 
 echo "go install google.golang.org/protobuf/cmd/protoc-gen-go"
```

Comment:
Maybe `"Executing: go install _______"` (etc)?

Reply:
Done. Let me know how you feel about the new echo-s